app.js

```diff
@@ -22,23 +22,21 @@ client.log(`Loaded database functions`)
 require('./library/client_events.js')(client)
 client.log(`Loaded client events`)
 
-let oldConnect = () => {
-  return new Promise(resolve => {
-    client.connectDB(db => resolve(db))
-  })
-}
-
 // weekly wipe at 12 am on monday
 cron.schedule('0 0 * * mon', () => {
   client.submitweek()
   client.clearWeekResults()
   client.log('Cleared weekly results')
 })
 
-connect('mongodb://localhost:27017/', 'pinano').then((userRepository, guildRepository) => {
+connect('mongodb://localhost:27017/', 'pinano').then(mongoManager => {
```

Comment:
@pianiststickman Take a look at this code here, which uses library/persistence. I think the way I have this set up, we can start running this code in production and incrementally port code over to using the new library. (I'd make a new PR with just the relevant commits from this branch.)

Reply:
Fantastic. I think porting recent feature changes to run on top of this should be fairly straight-forward. @ubermadness should probably take a look at this too, but at your leisure let's start moving code over to PracMan.